pom.xml

```diff
@@ -46,8 +46,8 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
```

Comment:
Is `8` valid here? I've always used `1.8` in the past and the [docs](https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html) seem to suggest it should be `1.8`.

Reply:
I think they support both notations because i can compile just fine the project, but to be extra sure it's better to use the format of the docs i agree.